docs/Reference/Rest_API/Rest.md

```diff
@@ -63,11 +63,22 @@ You can also use tools such as [Postman] or [cURL] to interact with Teku APIs.
 
 ## Enable the validator client API
 
-The [validator client API methods](https://ethereum.github.io/keymanager-APIs/) are enabled separately from the REST API methods.
+The [validator client API](../../HowTo/External-Signer/Manage-keys.md) allows you to call the [key manager API endpoints]((https://ethereum.github.io/keymanager-APIs/))
+and is enabled separately from the REST API methods.
 
 Enable the validator client API service from the command line by setting the
 [`--validator-api-enabled`](../CLI/CLI-Syntax.md#validator-api-enabled) command line option to `true`.
```

Comment:
```suggestion
Enable the validator client API service from the command line by including the
[`--validator-api-enabled`](../CLI/CLI-Syntax.md#validator-api-enabled) command line option.
```

As in your examples, it seems including the option is sufficient to enable it (without explicitly setting to `true`). If so, can you also update the option description?

Reply:
I updated here to match the example. I left the option description for now. We use both formats (`enabled` only and `enabled=true`) interchangeably for `metrics-enabled` and `rest-api-enabled` as well. Do you think we should we make them all consistent and leave off (or add) the `=true` for all of them, or leave as is?  Both ways function and I think it's intuitive enough for people to grasp, so I think it's okay to leave as is, but I can open another PR to update and clarify that both work if you think that makes the most sense.

Reply:
Ok, it's probably fine to leave as is. Thanks!